hs_core/templates/pages/genericresource.html

```diff
@@ -1494,10 +1494,10 @@ <h4 class="modal-title">Confirm files deletion</h4>
                 }
 
                 // If does not exist, add it
-                if (!exists && keyword[i] != "") {
+                if (!exists && keyword[i] != "" && keyword[i].length <= 100) {
```

Comment:
I was thinking of doing that. Then thought what else is there that could cause the update error since we have taken care of the duplicate keys and the length limit.